<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuTab</class>
 <widget class="QWidget" name="MenuTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_menu">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section { border: 1px solid #6c6c6c; }</string>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VAT %</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VAT Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="minimumHeight">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_menu_item">
       <property name="text">
        <string>Add Menu Item</string>
       </property>
       <property name="minimumHeight">
        <number>45</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>